spec/clearance/rack_session_spec.rb

```diff
@@ -2,21 +2,19 @@
 
 describe Clearance::RackSession do
   it 'injects a clearance session into the environment' do
-    expected_session = 'the session'
-    allow(expected_session).to receive(:add_cookie_to_headers)
-    allow(Clearance::Session).to receive(:new).and_return(expected_session)
     headers = { 'X-Roaring-Lobster' => 'Red' }
-
     app = Rack::Builder.new do
       use Clearance::RackSession
       run lambda { |env| Rack::Response.new(env[:clearance], 200, headers).finish }
     end
 
     env = Rack::MockRequest.env_for('/')
+    expected_session = 'the session'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.